pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -907,6 +907,12 @@ func GetClusterDomain() string {
 
 // MergeMaps merges two maps and returns the union
 func MergeMaps(a, b map[string]string) map[string]string {
+	if a == nil {
+		a = map[string]string{}
+	}
+	if b == nil {
```

Comment:
I think we don't need this.

Reply:
I dont't mind removing it, but keep in mind, that the next time this helper function will be used and b is nil, it will end up in the same situation as it is now - a panic.

@jiuker are you therefore sure, that we should make this change?